Cargo.toml

```diff
@@ -30,3 +30,4 @@ tokio-core = { version = "~0.1", optional = true }
 
 [dev-dependencies]
 version-sync = "~0.5"
+etherparse = "0.9.0"
```

Comment:
Style nitpick: Can you end each line with a newline, please? 😇

Reply:
No problem.